onboard/catkin_ws/src/cv/scripts/usb_camera.py

```diff
@@ -71,7 +71,9 @@ def run(self):
                 # Sleep loop to maintain frame rate
                 loop_rate.sleep()
         except Exception:
-            rospy.loginfo("Camera not found at channel {self.channel}")
+            rospy.logerr(f"Camera not found at channel {self.channel}")
+
+        rospy.spin()
```

Comment:
Why the spin? The only ways I can see this ending are if an error occurs. Don't you want it to stop in that case?

Reply:
I added the spin because the CI gave an error when trying to run `usb_camera.launch` that the process exited. I can also add `usb_camera.launch` to the block list to solve the CI error. Which do you prefer?

Reply:
I would lean towards adding it to a block list. CI isn't very helpful if it's rigged to always succeed. Maybe @nickchak21 would have a better idea on handling this though

Reply:
This isn't something I know much about, but I think I would prefer adding it to a block list as well. I don't generally see `rospy.spin()` used in this way.

Reply:
Added `usb_camera.launch` to blocklist and removed spin from `usb_camera.py`.